optuna_dashboard/_form_widget.py

```diff
@@ -152,6 +167,19 @@ def register_objective_form_widgets(
 def register_user_attr_form_widgets(
     study: optuna.Study, widgets: list[ObjectiveFormWidget]
 ) -> None:
+    if len(study.directions) != len(widgets):
+        raise ValueError("The length of actions must be the same with the number of objectives.")
+
```

Comment:
```suggestion
```
We don't need this validation because `len(study.directions)` and `len(widgets)` can be different for `register_user_attr_form_widgets`.